### PR TITLE
Added Copy Email Address to menu

### PR DESCRIPTION
**Pull request by manninglucas**
See issue #1318

### CODE REVIEW

File: js/contextMenus.js [hunk 1]
```diff
@@ -527,16 +527,30 @@ const copyLinkLocationMenuItem = (location) => {
   }
 }
 
+const copyEmailLocationMenuItem = (location) => {
+  return {
+    label: 'Copy Email address',
```
Comment (bbondy) [inline js/contextMenus.js]: nit capital on the 'a'

File: js/contextMenus.js [hunk 2]
```diff
@@ -527,16 +527,30 @@ const copyLinkLocationMenuItem = (location) => {
...
 
   if (nodeProps.href) {
     template.push(openInNewTabMenuItem(nodeProps.href, frame.get('isPrivate'), frame.get('partitionNumber')),
       openInNewPrivateTabMenuItem(nodeProps.href),
-      openInNewSessionTabMenuItem(nodeProps.href),
-      copyLinkLocationMenuItem(nodeProps.href),
-      CommonMenu.separatorMenuItem)
+      openInNewSessionTabMenuItem(nodeProps.href))
+
+    if (nodeProps.href.substring(0, 'mailto:'.length).toLowerCase() === 'mailto:') {
```
Comment (bbondy) [inline js/contextMenus.js]: nit: you can use .toLowerCase().startsWith instead of doing a substring.

**Comment by bbondy** [comment]
Just a couple tweaks, looks great thank you.
Please also rebase / squash the commits together via doing 
`git rebase -i master`

You can then just git push -f directly into this branch to update the pull request.

Thanks again!

**Comment by manninglucas** [comment]
Just realized the commits weren't squashed. Apologies for that. Hope everything else looks good :).

**Comment by manninglucas** [comment]
attempted to merge duplicate commits

**Comment by manninglucas** [comment]
@bbondy should this be PR be closed? I can resubmit if necessary.

**Comment by bbondy** [comment]
looks like there's still 8 commits, it should be rebased and squashed into a single commit please.  A new pull request is fine too if that's easier.

**Pull request closed without merging**